library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/api/ResourceLoader.java

```diff
@@ -50,6 +51,19 @@ public interface ResourceLoader {
 		return ResourceLoaderImpl.get(type);
 	}
 
+	/**
+	 * Gets the static resource manager for a given type.<br>
+	 * Static resources are different from regular resources in that they
+	 * are available as soon as the manager is requested and are (theoretically) immutable.<br>
+	 * It is <i>highly</i> recommended for mods to personally cache data they get from the provided manager in order to
+	 * prevent changes to userspace packs modifying data at runtime.
+	 * @param type the given resource type
+	 * @return the static resource manager instance
+	 */
+	static @NotNull ResourceManager getStaticResourceManager(@NotNull ResourceType type) {
```

Comment:
With the most recent commit, everything should be in order—bar the de-`static`ing of this method, which I'm opposed to, though if it's a sticking point for @LambdAurora I can change it. See  https://github.com/QuiltMC/quilt-standard-libraries/pull/321#discussion_r1264587712 for the reasoning on why that hasn't been done.

Reply:
Given that static resource managers are unrelated to the normal resource loaders, really - which handle non static data - I am also opposed to de-`static`ing this